### PR TITLE
About us

### PR DESCRIPTION
**Pull request by jroycodes**

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -7,18 +7,48 @@
     <link rel="stylesheet" href="scss/about.css" />
     <title>Document</title>
   </head>
+  <body>
+    <!-- SECTION 0  -->
```
Comment (codingknite) [inline about.html]: remove these comments in production

File: about.html [hunk 2]
```diff
@@ -7,18 +7,48 @@
...
-    <label class="menu">
-      <div></div>
-      <div></div>
-      <div></div>
-    </label>
-  </header>
+        <!-- Sidebar items -->
+        <div id="mySidebar" class="sidebar">
+          <div class="closebtn" onclick="closeNav()">&#128473;</div>
+          <div class="baritem">
```
Comment (codingknite) [inline about.html:49]: use html lists

```
<ul>
    <li>
        <a></a>
    </li>
</ul>
```

better semantics

File: about.html [hunk 3]
```diff
@@ -7,18 +7,48 @@
...
   </head>
+  <body>
+    <!-- SECTION 0  -->
+    <header>
+      <div class="logo">
+        <img src="Assets/Mobile/Logotest.png" alt="Healthcare logo" />
+      </div>
+
+      <!-- DESKTOP MENU -->
+      <div class="desktop_menu">
```
Comment (codingknite) [inline about.html]: use lists here too

File: about2.html [hunk 4]
```diff
@@ -0,0 +1,254 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="scss/styles.css" />
+    <link rel="stylesheet" href="scss/about2.css" />
```
Comment (codingknite) [inline about2.html:7]: why do you have two stylesheets in one page?

File: scss/about.css [hunk 5]
```diff
@@ -1,3 +1,115 @@
...
+  text-decoration: none;
+  color: #000000;
+}
+
+a:first-child {
+  color: #fc7f3a;
+}
+
+a:hover {
+  color: #fc7f3a;
```
Comment (codingknite) [inline scss/about.css:72]: this color has been used more than once, use a variable

File: scss/about2.css [hunk 6]
```diff
@@ -0,0 +1,178 @@
+@import url("https://fonts.googleapis.com/css2?family=IBM+Plex+Sans:wght@400;500;600;700&display=swap");
```
Comment (codingknite) [inline scss/about2.css:1]: import the font in the header...better for performance

File: scss/about2.css [hunk 7]
```diff
@@ -0,0 +1,178 @@
...
+  text-align: left;
+  font-size: 1.5rem;
+}
+.baritem a {
+  text-decoration: none;
+  color: #000000;
+}
+
+a:first-child {
+  color: #fc7f3a;
```
Comment (codingknite) [inline scss/about2.css:68]: use variable

**Review by codingknite — COMMENTED** [review]
make suggested changes, looks good overall

File: about.html (hunk 1 above)
Comment (jroycodes) [reply about.html]: Understood.

File: about2.html (hunk 4 above)
Comment (jroycodes) [reply about2.html:7]: Am re-using styles from the first stylesheet

**Pull request merged**